Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "iggy-py"
```

Comment:
@spetz @hubcio how should we call this library? Iggy is taken :( https://pypi.org/project/iggy/